test/common/autoscaling.go

```diff
@@ -0,0 +1,98 @@
+package common
+
+import (
+	"context"
+	rhmiv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/addon"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"time"
+)
+
+const (
+	hpaQuotaParam = "500"
+	hpaQuotaName  = "50 Million"
+)
+
+func TestAutoscaling(t TestingTB, ctx *TestingContext) {
+	quotaConfig, err := getQuotaConfig(t, ctx.Client)
+	if err != nil {
+		t.Fatalf("Error retrieving Quota: %v", err)
+	}
+
+	installation, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatal("couldn't get RHMI cr for quota test")
+	}
+	if installation == nil {
+		t.Fatalf("Got invalid rhmi CR: %v", installation)
+	}
+	initialHPAValue := installation.Spec.AutoscalingEnabled
+
+	// wait if stage no complete
+	startTime := time.Now()
+	endTime := startTime.Add(time.Minute * time.Duration(timeoutWaitingQuotachange))
+
+	for startTime.Before(endTime) {
+		startTime = time.Now()
+		if installation.Status.Stage == rhmiv1alpha1.CompleteStage {
+			break
+		}
+	}
+
+	//verify that the TOQuota value is set and that Quota is not set
+	//assuming this is run after installation
+	if installation.Status.Quota == "" {
+		t.Fatal("Expected '.status.quota' to contain a quota name after installation, but got an empty string")
+	}
+	if installation.Status.ToQuota != "" {
+		t.Fatalf("Expected .status.toQuota' to be set after installation to empty value, but got %s", installation.Status.ToQuota)
+	}
+
+	if installation.Status.Quota != quotaConfig.GetName() {
+		t.Fatalf("Expected quota name '%s' to match the expected value from the quota config map, but got: '%s'",
+			installation.Status.Quota, quotaConfig.GetName())
+	}
+	verifyConfiguration(t, ctx.Client, quotaConfig)
+
+	initialQuotaName := installation.Status.Quota
+	initialQuotaValue, found, err := addon.GetStringParameterByInstallType(context.TODO(), ctx.Client, rhmiv1alpha1.InstallationTypeManagedApi, RHMIOperatorNamespace, addon.QuotaParamName)
+	if !found {
+		t.Fatalf("failed to quota parameter '%s' from the parameter secret %v", addon.QuotaParamName, err)
+	}
+
+	// If hpa flag is false - set it to true
+	if initialHPAValue != true {
+		result, err := changeAutoscalingFlag(t, ctx.Client, installation, true)
+		if err != nil {
+			t.Fatalf("Error updating RHMI CR: %v with results of: %v", err, result)
+		}
+	}
+
+	// Check that quota has been changed
```

Comment:
In general I think the best flow of this test would be to:
1. Get initial quota
2. Set the autoscaling flag to true and expect quota to change to 50mil
3. Set the autoscaling flag to false and expect quota to bump back to whatever it was

However, we have to keep in mind that this might be a bit too heavy test to be listed as the happy path test as the quota bump will fail on prow due to lack of resources.

I would possibly introduce a quick check at the start of the test "inProw" (you can re-use existing function) and if it is in prow, skip test, otherwise, continue with test. 
I would also consider renaming the test to something else as we are not testing autoscaling here but the autoscaling flag.